### PR TITLE
[MFTF] add new AdminOpenCatalogProductPageActionGroup

### LINKED ISSUE #29941: [Issue] [MFTF] add new AdminOpenCatalogProductPageActionGroup
This issue is automatically created based on existing pull request: magento/magento2#29931: [MFTF] add new AdminOpenCatalogProductPageActionGroup

---------
This PR introduce `AdminOpenCatalogProductPageActionGroup` and makes  `GoToProductCatalogPageActionGroup` deprecated due to wrong naming

### PR DESCRIPTION
**Pull request by Usik2203**
This PR introduce `AdminOpenCatalogProductPageActionGroup` and makes  `GoToProductCatalogPageActionGroup` deprecated due to wrong naming

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/329

### Resolved issues:
1. [x] resolves magento/magento2#29941: [MFTF] add new AdminOpenCatalogProductPageActionGroup

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminOpenCatalogProductPageActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminOpenCatalogProductPageActionGroup">
+        <annotations>
+            <description>Open catalog product page.</description>
```
Comment (rogyar) [inline app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminOpenCatalogProductPageActionGroup.xml]: At first glance, I was confused with the definition "catalog product page in admin panel". After a couple of seconds, I've realized that we are talking about a page with the product grid. 
We may leave the name of the action group as it is. But I would suggest editing the description and have something like 

> Open page with product grid 

Or something like that. 

What do you think?

**Comment by Usik2203** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Comment by sidolov** [comment]
@magento create issue

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminOpenCatalogProductPageActionGroup.xml (hunk 1 above)
Comment (Usik2203) [reply app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminOpenCatalogProductPageActionGroup.xml]: **Done**

**Comment by Usik2203** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Comment by Usik2203** [comment]
Hi @slavvka 
Could i kindly ask you help me with failed EE and B2B test.
Maybe you could provide error message ?
Thanks.

**Comment by rogyar** [comment]
Hi @Usik2203. Have you tried to run the MFTF tests with your changes on EE/B2B setup locally? You might get a clue from this  attempt

**Comment by Usik2203** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Comment by Usik2203** [comment]
Hi @rogyar  
Fixed it by creating related PR 
Thanks.

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-8209](https://jira.corp.magento.com/browse/ENGCOM-8209) has been created to process this Pull Request
:eight_spoked_asterisk: @sidolov, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by gabrieldagama** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Comment by gabrieldagama** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

**Comment by gabrieldagama** [comment]
@magento run all tests

**Pull request merged**